clue.js

```diff
@@ -0,0 +1,316 @@
+}
+
+// OBJECTS FOR ALL THE WEAPONS
+
+const rope = {
+  name: 'Rope',
+  weight: 10,
+  id: "Rope",
+  image: './assets/rope.png'
+}
+
+const knife = {
+  name: 'Knife',
+  weight: 2,
+  id: "Knife",
+  image: './assets/knife.png'
+}
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 10,
+  id: "Candlestick",
+  image: './assets/candlestick.png'
+}
+
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 16,
+  id: "Dumbbell",
+  image: './assets/dumbell.png'
+}
+
+const poison = {
+  name: 'Bottole of Poison',
+  weight: 0.1,
+  id: "Poison",
+  image: './assets/poison.png'
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 5,
+  id: "Axe",
+  image: './assets/axe.png'
+}
+
+const bat = {
+  name: 'Bat',
+  weight: 8,
+  id: "Bat",
+  image: './assets/bat.png'
+}
+
+const trophy = {
+  name: 'Trophy',
+  weight: 8,
+  id: "Trophy",
+  image: './assets/trophy.png'
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 3,
+  id: "Pistol",
+  image: './assets/pistol.png'
+}
+
+// OBJECTS FOR ALL THE ROOMS
+
+const diningRoom = {
+  name: 'Dining Room',
+  image: './assets/dining.png'
+}
+
+const conservatory = {
+  name: 'Conservatory',
+  image: './assets/conservatory.png'
+}
+
+const kitchen = {
+  name: 'Kitchen',
+  image: './assets/kitchen.png'
+}
+
+const study = {
+  name: 'Study',
+  image: './assets/study.png'
+}
+
+const library = {
+  name: 'Library',
+  image: './assets/library.png'
+}
+
+const billiardRoom = {
+  name: 'Billiard Room',
+  image: './assets/ball.png'
+}
+
+const lounge = {
+  name: 'Lounge',
+  image: './assets/lounge.png'
+}
+
+const ballroom = {
+  name: 'Ballroom',
+  image: './assets/ballroom.png'
+}
+
+const hall = {
+  name: 'Hall',
+  image: './assets/hall.png'
+}
+
+const spa = {
+  name: 'Spa',
+  image: './assets/spa.png'
+}
+
+const livingRoom = {
+  name: 'Living Room',
+  image: './assets/livingroom.jpg'
+}
+
+const observatory = {
+  name: 'Observatory',
+  image: './assets/observatory.png'
+}
+
+const theater = {
+  name: 'Theater',
+  image: './assets/theater.png'
+}
+
+const guestHouse = {
+  name: 'Guest House',
+  image: './assets/guesthouse.png'
+}
+
+const patio = {
+  name: 'Patio',
+  image: './assets/patio.png'
+}
+
+//GROUPING ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS
+
+const suspects = [
+  mrGreen,
+  mrsWhite,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio
+]
+
+// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
+
+const randomSelector = array => {
+  return array[Math.floor(Math.random() * array.length)]
+}
+
+// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
+
+let mysteryobject = {
```

Comment:
Since you are not changing the structure of the object or is creating any other reference to mysterObject, consider changing this to a const.

Reply:
I've changed this into a const now! :)